roscala/src/test/scala/coop/rchain/roscala/VmSpec.scala

```diff
@@ -619,4 +619,59 @@ class VmSpec extends FlatSpec with Matchers {
 
     rtnCtxt.rslt shouldBe Fixnum(12)
   }
+
+  "OpLookupToReg" should "lookup binding in continuation when compiler can't find the compile-time binding for method." in {
+
+    /**
+      * This opcode sequence will be generated by the RBL
+      * compiler with the expression: (cat 1)
+      *
+      * litvec:
+      * 0:	{RequestExpr}
+      * 1:	'cat
+      * codevec:
+      * 0:	alloc 1
+      * 1:	lit 1,arg[0]
+      * 2:	lookup 1,trgt
+      * 3:	xmit/nxt 1
+      *
+      *
+      * this use case will go through the `absent` branch of `OpLookUpReg`, and the successful use case should be:
+      *
+      * (defProc (sbo& Top) (cat x) (+ x 1))
+      * (free [cat] (cat 1))     // this will print out `2`
+      *
+      * (free [〈id〉 …] 〈body〉) ⇒ result of 〈body〉
+      * The free form informs the compiler that the 〈id〉 are to be treated as free in 〈body〉;
+      * otherwise, the compiler will issue warnings that the 〈id〉 do not have a compile-time binding.
+      * And then it will be found in somewhere.
+      *
+      */
+    val codevec = Seq(
+      OpAlloc(n = 1),
+      OpImmediateLitToArg(literal = `1`, arg = 0),
+      OpLookupToReg(lit = `1`, reg = trgt),
```

Comment:
Please do `lit = 1` here. We use the backtick variables only for `literal`.

Reply:
Done